packages/fast-components-styles-msft/src/radio/index.ts

```diff
@@ -0,0 +1,96 @@
+import { IDesignSystem, withDesignSystemDefaults } from "../design-system";
+import { ComponentStyles, ComponentStyleSheet, ICSSRules } from "@microsoft/fast-jss-manager";
+import { IRadioClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import {
+    disabledContrast,
+    ensureNormalContrast,
+    hoverContrast,
+    normalContrast
+} from "../utilities/colors";
+
+/* tslint:disable:max-line-length */
+const styles: ComponentStyles<IRadioClassNameContract, IDesignSystem> = (config: IDesignSystem): ComponentStyleSheet<IRadioClassNameContract, IDesignSystem>  => {
+    const designSystem: IDesignSystem = withDesignSystemDefaults(config);
+    const backgroundColor: string = designSystem.backgroundColor;
+    const foregroundColor: string = ensureNormalContrast(config.contrast, designSystem.foregroundColor, designSystem.backgroundColor);
+    const radioColor: string = normalContrast(designSystem.contrast, foregroundColor, backgroundColor);
+    const radioHover: string = hoverContrast(designSystem.contrast, foregroundColor, backgroundColor);
+    const radioDisabled: string = disabledContrast(
+        designSystem.contrast,
+        foregroundColor,
+        backgroundColor
+    );
+
+    return {
+        radio: {
```

Comment:
These styles do not match radio styles, they look like checkbox. Also this should not be transparent it should use the background color.

Reply:
good catch. fixed